### PR TITLE
Amber/menu

### PR DESCRIPTION
**Pull request by amberwhizin**
yes?????

### CODE REVIEW

File: components/DropdownMenu.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  align-items: space-around;
+  height: 100%;
+`;
+
+const MenuLi = styled.li`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+  font-size: 30px;
```
Comment (briankeegan) [inline components/DropdownMenu.js]: Could we use rem?

Comment (amberwhizin) [reply components/DropdownMenu.js]: sure!

File: components/DropdownMenu.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+  font-size: 1.875rem;
+`;
+
+function DropDownMenu() {
+  const [isActive, setIsActive] = useState(false);
+  const handleClick = () => setIsActive(!isActive); //update or "set" aka (setIsActive)... click event from false to true (isActive)
```
Comment (briankeegan) [inline components/DropdownMenu.js]: how about toggleIsActive ?

Comment (briankeegan) [reply components/DropdownMenu.js]: [name nit]

**Review by briankeegan — APPROVED** [review]
Looks great. Lets wait on merging until the build is fixed.

File: components/DropdownMenu.js (hunk 2 above)
Comment (amberwhizin) [reply components/DropdownMenu.js]: good catch!

**Pull request merged**